docs/manage/data-forwarding/view-list-data-forwarding.md

```diff
@@ -7,16 +7,21 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 The page talks about viewing information about the data forwarding configured for your organization.
 
-1. <!--Kanso [**Classic UI**](/docs/get-started/sumo-logic-ui/). Kanso--> In the main Sumo Logic menu, select **Manage Data > Logs > Data Forwarding**. <!--Kanso <br/>[**New UI**](/docs/get-started/sumo-logic-ui-new/). In the top menu select **Configuration**, and then under **Logs** select **Data Forwarding**. You can also click the **Go To...** menu at the top of the screen and select **Data Forwarding**. Kanso--><br/><img src={useBaseUrl('/img/data-forwarding/data_forwarding.png')} alt="Data_Forwarding" width="1000"/>
+1. <!--Kanso [**Classic UI**](/docs/get-started/sumo-logic-ui/). Kanso--> In the main Sumo Logic menu, select **Manage Data > Logs > Data Forwarding**. <!--Kanso <br/>[**New UI**](/docs/get-started/sumo-logic-ui-new/). In the top menu select **Configuration**, and then under **Logs** select **Data Forwarding**. You can also click the **Go To...** menu at the top of the screen and select **Data Forwarding**. Kanso--><br/><img src={useBaseUrl('/img/data-forwarding/data_forwarding.png')} alt="Data_Forwarding" style={{border:'1px solid gray'}} width="800"/>
+1. Once you're in the **Data Forwarding** tab, apply the following settings:
+    * **Add a filter**. To refine the table results, utilize the **Add a filter** section located above the table. *AND* logic will be applied if you are filtering between different sections and *OR* logic will be applied if you are filtering between the same section.
```

Comment:
@JV0812 , if these edits look ok to you, could you please apply them to the other instances in this PR?

```suggestion
    * **Add a filter**. To refine the table results, utilize the **Add a Filter** section located above the table. *AND* logic will be applied if you are filtering between different sections and *OR* logic will be applied if you are filtering between the same section.
```

Reply:
@kimsauce the UI term is **Add a filter** so I have followed the same everywhere.

Reply:
Oops, I'm sorry about that - I accidentally forgot to apply my other changes to this sentence. It's fixed now for your review!

I also changed "Add a Filter" to "Add a filter" (lowercase F) since that's how it appears in the UI.

Reply:
And then if that sentence looks ok to you, would you be able to apply it to the other places in this PR where that same sentence appears?

Reply:
@kimsauce Thanks for the suggestion Kim, I will get this updated across other docs.